src/backend/distributed/planner/multi_logical_planner.c

```diff
@@ -554,6 +574,12 @@ ErrorIfSubqueryNotSupported(Query *subqueryTree)
 		errorDetail = "Subqueries with offset are not supported yet";
 	}
 
+	if (QueryReferencesView(subqueryTree))
+	{
+		preconditionsSatisfied = false;
```

Comment:
There are possible cases where it would work, my tests resulted in non-working cases to be norm. 

👍  on a comment. will do that.